packages/rlc-codegen/src/helpers/operationHelpers.ts

```diff
@@ -60,18 +60,22 @@ export function getPathParamDefinitions(
 }
 
 export function hasPagingOperations(model: RLCModel) {
-  for (const [path, details] of Object.entries(model.paths)) {
-    if (Boolean(details.annotations?.isPageable)) {
-      return true;
+  for (const details of Object.values(model.paths)) {
+    for (const method of Object.values(details.methods)) {
+      if (Boolean(method[0].annotations?.isPageable)) {
+        return true;
+      }
```

Comment:
why not use mode.pageInfo.hasPaging to decide whether we have paging operation ?

Reply:
Yes I think these two are equivalent and I could update accordingly.